setup.py

```diff
@@ -28,9 +28,12 @@
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
         "Topic :: Software Development :: Libraries :: Python Modules",
-        "Framework :: Django :: 2.2",
         "Framework :: Django :: 3.2",
+        "Framework :: Django :: 4.0",
+        "Framework :: Django :: 4.1",
+        "Framework :: Django :: 4.2",
```

Comment:
I'll leave it just because I see it listed in other packages
![Screenshot 2023-07-26 at 1 42 34 PM](https://github.com/caktus/django_bread/assets/87770895/b4c2f177-51c8-446f-99d2-95c3c029b725)